src/main/java/Duke.java

```diff
@@ -1,10 +1,78 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
```

Comment:
My sugeestion is restructuring the code according to CS2103T website requirement. Then you'll have more OO style code which is really better for you to manage your code and do the testing.